### DISCUSSION TITLE
Change the generated programming language

### DISCUSSION THREAD

**Discussion by Sipioteo** [GrapesJS/grapesjs#4911] [Q&A]
Hi, i was wondering if it's possible to customize the generated code.
I mean generating code in another language, not HTML, like you did for MJML.
I can't find any reference about this but i think it's possible in some how.

Can anyone help me?

**Comment by artf** [comment]
When you define/update component type you can define your own [`toHTML`](https://grapesjs.com/docs/api/component.html#tohtml) method and implement there your custom logic for the code.
